ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3632,4 +3632,26 @@ DRSM8X:
   - dryer_run_status_7
   - dryer_run_status_8
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING
+  
+AHAC:
+  description: "Tag to indicate an after hours activation method e.g. push button associated with this device."
+  is_abstract: true
+  opt_uses:  
```

Comment:
Omit if not used on all the new types defined.

Reply:
Ok, change made.